plugin/trino-tpch/src/main/java/io/trino/plugin/tpch/DoubleTypeMapping.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.tpch;
+
+import io.trino.spi.type.DecimalType;
+import io.trino.spi.type.DoubleType;
+import io.trino.spi.type.Type;
+
+import static java.util.Objects.requireNonNull;
+
+public enum DoubleTypeMapping
```

Comment:
Really the spec calls this type decimal, so I was originally thinking `DecimalTypeMapping`, but then I thought it might confuse Trino devs.  I don't really have any preference here.